sympy/integrals/tests/test_intpoly.py

```diff
@@ -514,3 +516,30 @@ def test_polytopes_intersecting_sides():
                    Point(4.203, 0.478))
     assert polytope_integrate(fig6, x**2 + x*y + y**2) ==\
         S(88161333955921)/(3*10**12)
+
+
+def test_max_degree():
+    polygon = Polygon(Point(0,0), Point(0,1), Point(1,1), Point(1,0))
```

Comment:
if you have to correct anything else you can add space after commas (and leave off the Point) and shorten the lines a few tests down.

Reply:
Sure! Will add spaces in the doctest where I copied the code from as well...